pkg/pillar/hypervisor/xen.go

```diff
@@ -464,7 +464,7 @@ func (ctx xenContext) Info(domainName string, domainID int) error {
 }
 
 func (ctx xenContext) LookupByName(domainName string, domainID int) (int, error) {
-	log.Debugf("xlDomid %s %d\n", domainName, domainID)
+	log.Infof("xlDomid %s %d\n", domainName, domainID)
```

Comment:
This gets called every few seconds forever, hence should remain at debug level to avoid gigabytes of logs.

Reply:
Changed.